main/ertp/guide/PursesAndPayments.md

```diff
@@ -0,0 +1,317 @@
+
+When you deposit assets into a `purse` or `payment`, they are added to
+whatever assets already exist there. So a 3 Quatloos deposit 
+into a `purse` with 8 Quatloos, results in a `purse` with 11 Quatloos. Similarly, 
+then withdrawing 6 Quatloos from that `purse` into a new `payment` leaves the `purse` 
+with 5 Quatloos and the new `payment` has 6 Quatloos. 
+
+When adding a `payment` to a `purse`, you must add the entire `payment`. To
+only add part of a `payment`, you must first use `payment.split()` 
+or `payment.splitMany()` to split the `payment` into two or more new `payments`.
+
+`mints` create entirely new digital assets and put them in a new `payment`. 
+You also create a `payment` by withdrawing assets from a `purse`, by splitting an 
+existing `payment`, or by combining multiple `payments` into one new one. Note 
+the `brand` of the new `payment` is the same as the associated `brand` of its originating `mint`, `purse` or `payment`. 
+
+You don't transfer assets directly from one `purse` to
+another `purse`. Instead, in ERTP you do something like these steps to send and receive assets. The actual send operation is up to you; ERTP does not 
+implement a way of sending object-containing messages between parties.
+- Send assets:
+  1. Withdraw assets described by an `amount`from a `purse`, creating a `payment`.
+  2. Send this `payment` to a recipient as a message.
+- Receive assets:
+  1. If you don't already have one, create a `purse` for the asset `brand`
+     you'll receive. 
+  2. Receive the message with the `payment`.
+  3. Deposit the `payment` in a `brand` appropriate `purse`. 
+     
+## Purses
+
+You change a purse's balance by calling either `deposit()` (to add assets) or
+`withdraw()` (to remove assets) on it. A purse can be empty, which for
+fungible assets means it has a value of 0. For non-fungible 
+assets, such as theatre tickets, it doesn't have any tickets.
+
+Unlike `payments`, `purses` are not meant to be sent to others. To transfer 
+digital assets, you should withdraw a `payment` from a `purse` and send 
+the `payment` to another party.
+
+You can create a *deposit facet* for a `purse`. Deposit facets are either sent to other parties or made publicly known. Any party can deposit a `payment` into the 
+deposit facet, which deposits it into its associated `purse`. However, no one can
+use a deposit facet to either make a withdrawal from its `purse` or get the `purse`'s balance. 
+
+If you have a deposit facet, you make a deposit to its associated `purse` by calling 
+`depositFacet.receive(payment);`. Note that you add a `payment` to a `purse` with a `deposit()` method, while you add a `payment` to a `depositFacet` with a `receive()` method. 
+
+The `payment` must be the same `brand` as what
+the associated `purse` object can contain. Otherwise it throws an error. 
+When sending a party a 
+deposit facet object, you should tell them what assets `brand` it accepts.
+
+![Purse methods](./assets/purse.svg)  
```

Comment:
The `(digital assets)` seems to be defining a purse, but a purse contains digital assets. I think digital assets should be an object in the diagram, connected by the line "can only contain". And Brand should go off digital assets, not the purse.